conf/messages.en

```diff
@@ -294,7 +294,7 @@ notEligible.reference.openingHours = Open 8am to 6pm, Monday to Friday (closed b
 notDeclarent.title = Use a different EORI number
 notDeclarent.paragraph1 = If you need to use a different EORI number you can {0}.
 notDeclarent.paragraph1Link = sign in using another Government Gateway ID
-notDeclarent.paragraph2 = If you need to use a different EORI number, log in using the account with the EORI number you need for this declaration. If you need to change your login details {0}.
+notDeclarent.paragraph2 = If there is a problem with your EORI number {0}.
```

Comment:
`If there is a problem with your EORI number GB123567` is it the correct content?

Reply:
The parameter is not the Eori number - its the content of the following link section.  I.e. "If there is a problem with your EORI number contact the EORI contact team."

Reply:
Thank you for the clarification